client/blocks/importer/wordpress/upgrade-plan/upgrade-plan-hosting-details.tsx

```diff
@@ -39,8 +45,37 @@ export const UpgradePlanHostingDetails = () => {
 		hostingProviderName !== 'Unknown' &&
 		hostingProviderName !== 'WordPress.com';
 
+	const navigateBack = () => {
+		const queryParams = new URLSearchParams( window.location.search );
+		queryParams.delete( 'siteSlug' );
+		queryParams.delete( 'showModal' );
+		// Navigate back to site picker keeping necessary the query params.
+		window.location.assign( `sitePicker?${ queryParams.toString() }` );
+	};
+
 	return (
 		<div className="import__upgrade-plan-hosting-details">
+			{ showModal && (
+				<ConfirmModal
+					compact={ false }
+					title={ translate( 'Migration sounds daunting? It shouldn’t be!' ) }
+					confirmText="Take the deal"
+					cancelText="No, thanks"
+					onClose={ navigateBack }
+					onConfirm={ () => {} }
+				>
+					<p>
+						{ translate(
+							'Subscribe to the Creator plan now, and get a complimentary migration service (normally $500) to move magical-pinneaples.fun to WordPress.com.'
```

Comment:
I know this is just a placeholder, but let's try to have the site name instead of this, so we can directly have the right translations done :)

Reply:
I was planning to handle it soon, but can be now too :) addressed.